packages/notifications/src/lib/notifications/store/store.initial-state.ts

```diff
@@ -1,4 +1,5 @@
 import { NotificationsState } from './store.types';
+import { NOTIFICATIONS_INITIAL_VALUE } from './notifications/notifications.initial-state';
```

Comment:
Please be careful when using `git stash`!

Reply:
Deze zit er nog steeds in?!

Reply:
ik kan de NOTIFICATIONS_INITIAL_VALUE toch niet weg doen, wat dan is er een error dat deze niet kan gevonden worden.  wat is uw voorstel?